src/utils/facadeLocator.js

```diff
@@ -0,0 +1,23 @@
+function stripSlashes(path) {
+	return path.replace(/^(\/+)|(\/+)$/g, '');
+}
+
+const setupFacadeLocator = (app) => {
+	app.facadeDict = {};
```

Comment:
I would like if we use the app.set('facadeDict', {}) and app.get('facadeDict'); for it.

Reply:
Convert setupFacadeLocator to class or use a const outside of setupFacadeLocator but in this file to hold the facades ..clear the app from data.